freezeyt/freezer.py

```diff
@@ -448,10 +438,14 @@ def start_response(
             if value is not None:
                 raise value
 
-        if self.status_handlers.get(status[:3]):
+        if self.status_handlers.get(status[:3]): # handle particular status from configuration
             status_handler = self.status_handlers.get(status[:3])
-        elif self.status_handlers.get(status[0] + 'xx'):
+        elif self.status_handlers.get(status[0] + 'xx'): # handle group statuses from configuration
             status_handler = self.status_handlers.get(status[0] + 'xx')
+        elif status.startswith('200'): # default for status 200
+            status_handler = freezeyt.status_handlers.save
+        elif status[0] in '12345': # default for groups 1xx, 2xx, ...
```

Comment:
We should get response in some standardized way (RFC I suppose), if we check the first char in known range of statuses that should be fine...